frontend/components/vault/VaultsTable.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+    <Loading :is-loading="showLoadingOverlay" :error="error">
+        <Table
+            ref="tableContainer"
+            :data-source="vaultTransactions"
+            :columns="columns"
+            :pagination="{ pageSize: numberOfRowsPerPage, hideOnSinglePage: true }"
+            :row-class-name="getRowClassNames"
+            :row-key="record => record.id"
+            :custom-row="customRowEvents"
+            :get-popup-container="() => $el"
+            :locale="{ emptyText: 'There are currently no vaults at risk.' }"
+            class="VaultsTable relative overflow-visible"
+        >
+            <div slot="collateralAmount" slot-scope="collateralAmount, record">
+                <format-currency
+                    v-if="collateralAmount && record.collateralType"
+                    :value="collateralAmount"
+                    :currency="record.collateralType"
+                />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+            <div slot="proximityToLiquidation" slot-scope="proximityToLiquidation">
+                <template v-if="proximityToLiquidation">
+                    <format-percentage :value="proximityToLiquidation" />
```

Comment:
Didn't you use `AnimatedNumber` to format the same number in the `Vault` table? I think they would be then displayed differently, right?